lib/phoenix_live_view/upload_config.ex

```diff
@@ -270,6 +273,26 @@ defmodule Phoenix.LiveView.UploadConfig do
           nil
       end
 
+    writer =
+      case Keyword.fetch(opts, :writer) do
+        {:ok, func} when is_function(func) ->
+          func
+
+        {:ok, other} ->
+          raise ArgumentError, """
```

Comment:
I am a bit confused. Here we say it has to be a function but in the writer we say it is either a module or a function. Which one is it?

Reply:
it is both. I will update!

Reply:
we only accept a 3-arity function now :)